site/index.html

```diff
@@ -35,7 +40,13 @@ <h4 class="tag">Declarative, statically typed code.</h4>
       </div>
    </div>
 </div>
-<br><br class="hidden-xs hidden-sm"><br class="hidden-xs hidden-sm">
+
+<br>
+<br class="hidden-sm">
+<br class="hidden-sm">
+<br class="hidden-xs hidden-sm">
+<br class="hidden-xs hidden-sm">
```

Comment:
Here I added some extra spacing, as explained above. So before it was 3 brs, now it is 5. Notice that just one happens on any screen size, other 2 happen only when screen is > sm, and other 2 happen only when screen > xs. So, smaller the screen, less spacing, which is what we want.